near/omni-types/src/lib.rs

```diff
@@ -233,6 +249,14 @@ impl OmniAddress {
                 Self::new_from_evm_address(chain_kind, Self::to_evm_address(address)?)
             }
             ChainKind::Near => Ok(Self::Near(Self::to_near_account_id(address)?)),
+            ChainKind::Btc => Ok(Self::Btc(
+                String::from_utf8(address.to_vec())
+                    .map_err(|e| format!("Invalid BTC address: {e}"))?,
+            )),
+            ChainKind::Zcash => Ok(Self::Zcash(
```

Comment:
We use short tickers, so `Zec` align more with other prefixes. But personally I don't like the `Zec` naming because it doesn't sound like Zcash and it looks like a forced truncation.
Lets see what others thinks about that. 
CC @r-near @kiseln @frolvanya